build/Targets/VSL.Versions.targets

```diff
@@ -61,14 +61,13 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <VSSemanticVersion>15.0.1</VSSemanticVersion>
+    <VSSemanticVersion>15.0</VSSemanticVersion>
```

Comment:
Nuget versions has 3 parts. `Major`.`Minor`.`Patch`. Later, the property is updated to append the BuildNumber and the Number of build for the day. Example version would look like this `15.0.6142703`. Look at the above location where I have the sample output.